include/devicetree/dma.h

```diff
@@ -101,6 +101,74 @@ extern "C" {
 #define DT_DMAS_LABEL_BY_NAME(node_id, name) \
 	DT_PROP(DT_PHANDLE_BY_NAME(node_id, dmas, name), label)
 
+/**
+ * @brief Get the node identifier for the DMA controller from a
+ *        dmas property at an index
+ *
+ * Example devicetree fragment:
+ *
+ *     dma1: dma@... { ... };
+ *
+ *     dma2: dma@... { ... };
+ *
+ *     n: node {
+ *		dmas = <&dma1 1 2 0x400 0x3>,
+ *			<&dma2 6 3 0x404 0x5>;
+ *		dma-names = "tx", "rx";
```

Comment:
```suggestion
```

Would like to see this removed just so irrelevant properties don't potentially confuse users.

Reply:
Done.